### PR TITLE
Volunteer Opportunities Page (design only visible to logged in user)

### PR DESCRIPTION
**Pull request by ankoanju**
Volunteer opportunities page. This design has the sidenavbar & assumes logged in user. 
@lisanag please edit the sidenavbar icons to the volunteer dashboard icons.

This file will/needs to be edited more later in order to make it fully responsive. 
Needs:
 - the first section elements (filters, search bar) needs to be responsive.
 - open a full list of filters once the 'more' filter/icon is clicked.

Done:
 - cards displaying opportunities are fully responsive.

### CODE REVIEW

**Review by anavianm — APPROVED** [review]
Looks good! but I don't see any files? is that a github bug

File: users/templates/users/volunteeropp.html [hunk 1]
```diff
@@ -0,0 +1,714 @@
...
+	<link rel="stylesheet" type="text/css" href="../template.css">
+	<link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
+
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
+
+
+	<title> Volunteer Opportunities </title>
+
+	<style>
```
Comment (mykyta-protsenko) [inline users/templates/users/volunteeropp.html:15]: Does it make sense to pull styles to a separate CSS file? And maybe reconcile it with the CSS file @lisanag created?

(Not urgent, though, can be done as a separate PR.)

**Review by mykyta-protsenko — APPROVED** [review]

**Review by anavianm — APPROVED** [review]
Looks good! We might put the CSS into an external style sheet later but otherwise its amazing!

**Comment by lisanag** [comment]
Great job! Very minor but I think the filter icons should have a cursor pointer on hover.

**Comment by lisanag** [comment]
> Great job! Very minor but I think the filter icons should have a cursor pointer on hover.

Oh no I accidentally closedit I will reopen it, trying to figure out pull requests still

**Review by lisanag — APPROVED** [review]

**Pull request merged**